src/components/tiles/text/toolbar/text-toolbar-registration.tsx

```diff
@@ -36,14 +36,17 @@ interface IGenericTextToolbarButtonProps {
 
 function GenericTextToolbarButton({name, title, keyHint, Icon, slateType, toggleFunc}: IGenericTextToolbarButtonProps) {
   const editor = useSlate();
+  const chipSelected = isHighlightChipSelected(editor);
   const selected = editor.isMarkActive(slateType) || editor.isElementActive(slateType);
+  const disabled = chipSelected;
```

Comment:
I think what you are doing here is disabling all of the other buttons when there is a selected chip. This works for any button that uses GenericTextToolbarButton. However the `LinkButton` doesn't use GenericTextToolbarButton, so it won't get disabled. 

Since you've built up other PRs on this one you don't need to fix it, but it'd be better to find another way to disable all of the buttons rather than modify the GenericTextToolbarButton.